zone/mob.h

```diff
@@ -1839,6 +1866,18 @@ class Mob : public Entity {
 	EQ::InventoryProfile m_inv;
 	std::shared_ptr<HealRotation> m_target_of_heal_rotation;
 	bool m_manual_follow;
+
+	void SetHeroicStrBonuses();
+
+	void SetHeroicStaBonuses();
+
+	void SetHeroicAgiBonuses();
+
+	void SetHeroicDexBonuses();
+
+	void SetHeroicIntBonuses();
+
+	void SetHeroicWisBonuses();
```

Comment:
Why all the newlines between?

Reply:
guess that's how clion created them, fixed,